### PR TITLE
always return formatted results from runner

### PR DESCRIPTION
**Pull request by brendankenny**
Besides just being more consistent, this also allows printed `--output json` results of the raw audit results, even if no aggregations are given in the config.

Also a good time to delete `lighthouse-core/aggregations/index.js` since that was an entire file for a map statement :)

### CODE REVIEW

File: lighthouse-cli/printer.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ function createOutput(results, outputMode) {
 
   // JSON report.
   if (outputMode === 'json') {
-    return JSON.stringify({audits: results.audits, aggregations: results.aggregations}, null, 2);
+    return JSON.stringify(results, null, 2);
```
Comment (brendankenny) [inline lighthouse-cli/printer.js]: json print fill results, including url and initialUrl

**Comment by paulirish** [comment]
seems good. In the case of no aggregations, we were just returning nothing?

**Comment by brendankenny** [comment]
> In the case of no aggregations, we were just returning nothing?

the array of audit results were being returned directly, and the json printer was printing `{audits: results.audits, aggregations: results.aggregations}`, so it was printing just an empty `{}`

File: lighthouse-cli/printer.js (hunk 1 above)
Comment (paulirish) [inline lighthouse-cli/printer.js]: lgtm on that. 

can you confirm we have test coverage for the structure of the JSON output? 
does the coverage extend to include the CLI/printer code?

**Comment by paulirish** [comment]
lgtm pending the question about test coverage.

**Comment by brendankenny** [comment]
sweet json output test added, fully end-to-end (of module, not cli)

**Pull request merged**